build/configs/imxrt1050-evk/loadable_elf_apps/Make.defs

```diff
@@ -139,7 +139,7 @@ AFLAGS = $(CFLAGS) -D__ASSEMBLY__
 CELFFLAGS = $(CFLAGS) -mlong-calls # --target1-abs
 CXXELFFLAGS = $(CXXFLAGS) -mlong-calls # --target1-abs
 
-LDELFFLAGS = -r -e main
+LDELFFLAGS = -r -e main -s
```

Comment:
Better to put inside a flag and if dynamic linking is enabled, then this flag should n't be there. Account  these.